mongodbatlas/resource_mongodbatlas_cluster.go

```diff
@@ -1588,3 +1612,27 @@ func clusterAdvancedConfigurationSchema() *schema.Schema {
 		},
 	}
 }
+
+func updateCluster(ctx context.Context, conn *matlas.Client, request *matlas.Cluster, projectID, name string) (*matlas.Cluster, *matlas.Response, error) {
+	cluster, resp, err := conn.Clusters.Update(ctx, projectID, name, request)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	stateConf := &resource.StateChangeConf{
+		Pending:    []string{"CREATING", "UPDATING", "REPAIRING"},
+		Target:     []string{"IDLE"},
```

Comment:
[q] given you use this function to pause a cluster wouldn't you need to add `PAUSED` as a goal state here or risk staying on an infinite loop?

Reply:
Just tested again, and it seems it doesn't have a state called `PAUSED` but it shows paused as true. Here is a response of my recent test

```
"name": "test-acc-oceq6iwemq",
 "numShards": 1,
 "paused": true,

...

 "stateName": "IDLE",
```